Makefile

```diff
@@ -1,14 +1,14 @@
 .PHONY: clean dev all check checkstatic unittests test phpdev javascript
 
 all: VERSION javascript
-	composer install --no-dev
+	composer install --no-dev --prefer-dist
```

Comment:
```suggestion
	composer install --no-dev
```

Reply:
dist is already the default for stable versions of packages and there isn't sufficient justification to override the composer defaults in this PR.